downstream/modules/platform/proc-aap-hub-backup.adoc

```diff
@@ -0,0 +1,26 @@
+[id="aap-hub-backup"]
+
+= Backing up the {HubNameStart} deployment
+
+[role=_abstract]
+Use this procedure to back up a deployment of the hub, including all hosted Ansible content.
+
+.Prerequisites
+
+* You must be authenticated with an Openshift cluster.
+* The {OperatorPlatform} has been deployed to the cluster.
+* The {PlatformName} is deployed to using the operator.
```

Comment:
Same thing here.  This renders as:
```
The Red Hat Ansible Automation Platform is deployed to using the operator.
```

It should instead read:
```
Automation Hub is deployed to using the AAP Operator.
```

Reply:
Fixed.